Chapters/4-3-Chapter25.md

```diff
@@ -0,0 +1,196 @@
+<!--
+  Annui flees to Rudjan?
+
+  The chapter starts at the edge of Heiko ruins: a massive crater of rubble.
+
+ -->
+
+## Chapter 25
+
+The rubble was everywhere. I would never have believed that something so simple could haunt me as much as that path toward Ara. Everywhere we walked, our journey was punctuated with sharp edges and unstable footing. We could barely see our feet but for the faintly ambient glow of Ara bouncing through the neverending cloud of dust, and the small lantern that Pomri had to continuously wipe clean. I was glad of her foresight.
```

Comment:
So there was a film I saw ages ago, it feature the bombing of Nagasaki. I forgot what it's called, but I highly recommend you watch it. Don't eat before, you might lose some food.